roles/kuryr/tasks/master.yaml

```
@@ -1,4 +1,11 @@
 ---
+- name: Ensure project exists
+  oc_project:
+    name: "{{ kuryr_namespace }}"
+    state: present
+    node_selector:
+      - ""
```

This causes indentation errors:
 ```suggestion
    - ""
```
/lgtm cancel

---

Strange, I took the same idea as openshift-sdn: https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/openshift_sdn/tasks/main.yml#L2-L7